docs/compendium/substitute.md

```diff
@@ -1,31 +1,277 @@
-# Substitute compendia
+# Substitute two compendia
 
-introduction...
+Substitution is the combination of an base ERC and an overlay ERC.
+A user can choose files from the overlay ERC that will replace files of the base ERC or will be uniquely added.
 
 ## Create substitution
 
+`Create substitution` will produce a new ERC with metadata, saved to mongoDB.
```

Comment:
The name is "MongoDB".

Also, please clarify the accepted POST payload, see https://github.com/o2r-project/o2r-web-api/issues/51

Reply:
Similar sentence as here: https://github.com/o2r-project/o2r-web-api/commit/692b27f2ca66e64a475a4fc70fb76c16ec990ee5

Reply:
done